app/pages/organizations/organization-container.jsx

```diff
@@ -0,0 +1,199 @@
+      error: null,
+      fetching: false,
+      organization: null,
+      organizationAvatar: {},
+      organizationBackground: {},
+      organizationRoles: [],
+      organizationPages: [],
+      organizationProjects: [],
+      view: false
+    };
+
+    this.toggleCollaboratorView = this.toggleCollaboratorView.bind(this);
+  }
+
+  componentDidMount() {
+    if (this.context.initialLoadComplete) {
+      this.fetchOrganization(this.props.params.name, this.props.params.owner);
+    }
+  }
+
+  componentWillReceiveProps(nextProps, nextContext) {
+    const noOrgAfterLoad = nextContext.initialLoadComplete && this.state.organization === null;
+
+    if (nextProps.params.name !== this.props.params.name ||
+      nextProps.params.owner !== this.props.params.owner ||
+      noOrgAfterLoad) {
+      if (!this.state.fetching) {
+        this.fetchOrganization(nextProps.params.name, nextProps.params.owner);
+      }
+    }
+  }
+
+  isCollaborator() {
+    if (!this.context.user) {
+      return false;
+    }
+
+    const collaboratorRoles = this.state.organizationRoles.filter(role =>
+      role.roles.includes('collaborator') || role.roles.includes('owner'));
+    return collaboratorRoles.some(role => role.links.owner.id === this.context.user.id);
+  }
+
+  checkView() {
+    if (this.state.organization && this.state.organization.listed) {
+      return true;
+    } else if (isAdmin()) {
+      return true;
+    } else {
+      return this.isCollaborator();
+    }
+  }
+
+  toggleCollaboratorView() {
+    const newView = !this.state.collaboratorView;
+    this.setState({ collaboratorView: newView });
+    this.fetchProjects(this.state.organization, newView);
+  }
+
+  fetchProjects(organization, collaboratorView = false) {
+    const query = { cards: true };
+    if (!collaboratorView) {
+      query.launch_approved = true;
+    }
+
+    organization.get('projects', query)
+      .then(organizationProjects => this.setState({ organizationProjects }))
+      .catch(error => console.error('error loading projects', error)); // eslint-disable-line no-console
+  }
+
+  fetchOrganization(name, owner) {
+    if (!name || !owner) {
+      return;
+    }
+
+    this.setState({ fetching: true, error: null });
+
+    const slug = `${owner}/${name}`;
+
+    apiClient.type('organizations').get({ slug, include: 'avatar,background,organization_roles,pages' })
+      .then(([organization]) => {
+        this.setState({ organization });
+
+        if (organization) {
+          const awaitAvatar = apiClient.type('avatars').get(organization.links.avatar.id)
+            .catch(error => console.error('error loading avatar', error)); // eslint-disable-line no-console
+          const awaitBackground = apiClient.type('backgrounds').get(organization.links.background.id)
+            .catch(error => console.error('error loading background', error)); // eslint-disable-line no-console
+          const awaitRoles = organization.get('organization_roles')
```

Comment:
Can this be retrieved out the client cache since it was a part of the include instead?